mlos_bench/mlos_bench/optimizers/convert_configspace.py

```diff
@@ -76,24 +79,51 @@ def _tunable_to_configspace(
                 meta=meta)
         })
 
+    distribution: Union[Uniform, Normal, Beta, None] = None
+    if tunable.distribution == "uniform":
```

Comment:
```suggestion
    if tunable.distribution == "uniform":
```

Enums instead of static strings?

Reply:
the type is `Literal["uniform", "normal", "beta"]` so I guess we're ok